ui/app/scripts/services/AuthFactory.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+/*global app: false */
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['app' is defined but never used.](https://www.codacy.com/app/HiP-App/HiPCMS/file/2288686128/issues/source?bid=3144120&fileBranchId=3183264#l3)